### PR TITLE
Add coq-hierarchy-builder for Coq 8.10 and 8.11

### PR DESCRIPTION
**Pull request by gares**

### CODE REVIEW

File: released/packages/coq-hierarchy-builder/coq-hierarchy-builder.0.9.0/opam [hunk 1]
```diff
@@ -0,0 +1,24 @@
+opam-version: "2.0"
+name: "coq-hierarchy-builder"
+version: "dev"
```
Comment (palmskog) [inline released/packages/coq-hierarchy-builder/coq-hierarchy-builder.0.9.0/opam]:
```suggestion
```
The explicit version name conflicts with the version name defined by the path.

File: released/packages/coq-hierarchy-builder/coq-hierarchy-builder.0.9.0/opam [hunk 2]
```diff
@@ -0,0 +1,24 @@
+opam-version: "2.0"
+name: "coq-hierarchy-builder"
```
Comment (palmskog) [inline released/packages/coq-hierarchy-builder/coq-hierarchy-builder.0.9.0/opam]:
```suggestion
```
This is redundant.

File: released/packages/coq-hierarchy-builder/coq-hierarchy-builder.0.9.0/opam [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+version: "dev"
+maintainer: "Enrico Tassi <enrico.tassi@inria.fr>"
+authors: [ "Cyril Cohen" "Kazuhiko Sakaguchi" "Enrico Tassi" ]
+license: "MIT"
+homepage: "https://github.com/math-comp/hierarchy-builder"
+bug-reports: "https://github.com/math-comp/hierarchy-builder/issues"
+dev-repo: "git+https://github.com/math-comp/hierarchy-builder"
+
+build: [ make ]
```
Comment (palmskog) [inline released/packages/coq-hierarchy-builder/coq-hierarchy-builder.0.9.0/opam]:
```suggestion
build: [ make "-j%{jobs}%" ]
```
Is there a reason for not building in parallel?

Comment (gares) [reply released/packages/coq-hierarchy-builder/coq-hierarchy-builder.0.9.0/opam]: It takes only a few seconds, but I believe it works

**Comment by gares** [comment]
Thanks for the comments, I made a new PR taking the remarks into account

**Comment by gares** [comment]
Do you know other projects that could claim HB?
The main problem is that all commands are named `HB.something` so changing the logical path would make things less consistent...

**Pull request merged**